docs/service-contraints.md

```diff
@@ -0,0 +1,80 @@
+# Distbench node attributes and service constraints support
+
+Distbench node managers can have different attributes as part of their description, as
```

Comment:
You should introduce the purpose or goal that the feature enables, before explaining how this feature solves it.